src/utils/countryCurrency/getExchange.js

```diff
@@ -0,0 +1,6 @@
+import countryExchange from './countryExchange.json';
+
+export const getExchangeValue = (curreny) => {
```

Comment:
I understood we use these values only for calculating the minimum amount to donate, but we 
- should make sure nobody else is using these values in the frontend for the purpose of showing real donation amounts in other currencies!
- make sure exchange rates get adapted from time to time

So maybe we can add some comment in this file to describe the purpose and constraints? Or we rewrite this function to `getMinimumAmountForCurrency()` containing the calculation `Math.round(exchange.value * 2)`, so nobody ever gets the idea of using `getExchangeValue()` for any other purpose.

Reply:
I agree with Norbert. When I shared the sheet with @captainamiedi  I asked him to already multiply the value in the json file.

Reply:
Thanks @norbertschuler for these feedback. i have made changes to these effect. Kindly review and give feedback.